bokehjs/test/models/tools/edit/point_draw_tool.ts

```diff
@@ -0,0 +1,212 @@
+const {expect} = require("chai");
+const utils = require("../../../utils");
+const sinon = require('sinon');
+
+const {Keys} = utils.require("core/dom");
+const {create_1d_hit_test_result, create_hit_test_result} = utils.require("core/hittest");
+
+const {PointDrawTool, PointDrawToolView} = utils.require("models/tools/edit/point_draw_tool");
+const {Range1d} = utils.require("models/ranges/range1d");
+const {Circle} = utils.require("models/glyphs/circle");
+const {GlyphRenderer} = utils.require("models/renderers/glyph_renderer");
+const {ColumnDataSource} = utils.require("models/sources/column_data_source");
+const {Plot} = utils.require("models/plots/plot");
+const {Toolbar} = utils.require("models/tools/toolbar");
```

Comment:
You will see this in other files I listed, but those can be rewritten as imports, so e.g.:
```typescript
import * as sinon from "sinon"
import {PointDrawTool, PointDrawToolView} from "models/tools/edit/point_draw_tool"
```
Helper function `util.require()` isn't necessary anymore, because typescript does module mapping and knows how to resolve bokehjs' modules (`paths` in `tests/tsconfig.json` governs this).

Reply:
Working on this now, just had to add the sinon typings to the requirements.

Reply:
That's good.